app/api/helpers/mail.py

```diff
@@ -449,3 +449,29 @@ def send_order_cancel_email(order):
             app_name=get_settings()['app_name'],
         ),
     )
+
+
+def send_email_to_moderator(video_stream, video_moderator, user):
```

Comment:
Just receive video_moderator in the params. You can get stream and user from video_moderator